### PR TITLE
#215 Create app icon && #218 Add "About Us" info

### PR DESCRIPTION
**Pull request by reibek**
* vectorized logo and used as app icon
* slightly changed home page font to "Roboto" to mach Material Design rules
* added footer with link to "About Us" page
* added page "About Us"
* some minor design changes

### CODE REVIEW

File: WebApp/src/components/AboutUsPage/AboutUsPage.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+          We are students from Faculty of informatics and management science. We are trying to create an unique app for students of University of Žilina (firstly just our faculty), 
+          where they can change their timetables and each subjects. The whole idea is that students don´t have to seek for someone who want to change their subject in timetable too, 
+          but they can easily change it via the app. It should be the place where you can make changes, and system will find you another person who wants reverse change.
+        </p>
+      </div>
+    );
+  }
+}
+
+export default connect()(AboutUsPage);
```
Comment (mdomdo0) [inline WebApp/src/components/AboutUsPage/AboutUsPage.js]: fine

File: WebApp/src/components/HomePage/HomePage.js [hunk 2]
```diff
@@ -47,13 +43,18 @@ class HomePage extends Component {
...
+              <NavLink to="/register" activeClassName="PageSwitcher__Item--Active" className="PageSwitcher__Item">Registrácia</NavLink>
             </div>
             <Route exact path="/" component={LoginPage}>
             </Route>
             <Route path="/register" component={RegisterPage}>
             </Route>
+            <Route path="/aboutus" component={AboutUsPage}>
+            </Route>
+            <div className="HomeFooter">
+              <NavLink exact to="/aboutus" className="FormField__Link">O nás</NavLink>
```
Comment (mdomdo0) [inline WebApp/src/components/HomePage/HomePage.js]: ok

File: WebApp/src/components/RegisterPage/RegisterPage.js [hunk 3]
```diff
@@ -102,7 +102,7 @@ class RegisterPage extends Component {
 
 
   handleChange = event => {
-    event.preventDefault();
+    //event.preventDefault();
```
Comment (mdomdo0) [inline WebApp/src/components/RegisterPage/RegisterPage.js]: why commented

Comment (reibek) [reply WebApp/src/components/RegisterPage/RegisterPage.js]: It prevented chceckbox to be checked or unchecked on first try.

**Pull request merged**